deploy/kubeturbo-operator/deploy/crds/charts_v1alpha1_kubeturbo_cr.yaml

```diff
@@ -14,6 +14,9 @@ spec:
     version: "8.1"
     turboServer: https://Turbo_server_URL
 
+  # URL format for Authenticated and Non-Authenticated
+  #   proxy: "http://username:password@ip:port or http://ip:port"  
```

Comment:
`proxyserver:proxyport` ?

Reply:
are you suggesting to use those terms?

Reply:
Yes, to make them consistent across the code base.